### PR TITLE
fix: generate and upload chart of performance metrics data

### LINKED ISSUE #3087: CICD: Add a comment with detailed performance metrics data in every PR
### Feature to Enhance

In the `Performance E2E tests` CI, add a step that will upload the `timeline-metrics.json` file and, if possible, a generated chart image, to an external long-term storage. Link the artifacts in a comment in the PR

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
## Description

This PR adds a TS script that generates a chart of the data from the Performance E2E tests. Two charts - with grouped and non-grouped pod data, and the data JSON file are archived and uploaded as an artifact at the end of the performance E2E workflow.

### Related Issues

* Closes #3087
* Related to #

### Pull request (PR) checklist

* \[ ] This PR added tests (unit, integration, and/or end-to-end)
* \[ ] This PR updated documentation
* \[ ] This PR added no TODOs or commented out code
* \[ ] This PR has no breaking changes
* \[ ] Any technical debt has been documented as a separate issue and linked to this PR
* \[ ] Any `package.json` changes have been explained to and approved by a repository manager
* \[ ] All related issues have been linked to this PR
* \[ ] All changes in this PR are included in the description
* \[ ] When this PR merges the commits will be squashed and the title will be used as the commit message, the 'commit message guidelines' below have been followed

### Testing

* \[ ] This PR added unit tests
* \[ ] This PR added integration/end-to-end tests
* \[ ] These changes required manual testing that is documented below
* \[ ] Anything not tested is documented

The following manual testing was done:

* TBD

The following was not tested:

* TBD

<details>
<summary>
Commit message guidelines
</summary>
We use 'Conventional Commits' to ensure that our commit messages are easy to read, follow a consistent format, and for automated release note generation. Please follow the guidelines below when writing your commit messages:

1. BREAKING CHANGE: a commit that has a footer BREAKING CHANGE:, or appends a ! after the type/scope, introduces a breaking API change (correlating with MAJOR in Semantic Versioning). A BREAKING CHANGE can be part of commits of any type.  NOTE: currently breaking changes will only bump the MAJOR version.
2. The title is prefixed with one of the following:

| Prefix    | Description                                         | Semantic Version Update | Captured in Release Notes |
|-----------|-----------------------------------------------------|-------------------------|---------------------------|
| feat:     | a new feature                                       | MINOR                   | Yes                       |
| fix:      | a bug fix                                           | PATCH                   | Yes                       |
| perf:     | performance                                         | PATCH                   | Yes                       |
| refactor: | code change that isn't feature or fix               | none                    | No                        |
| test:     | adding missing tests                                | none                    | No                        |
| docs:     | changes to documentation                            | none                    | Yes                        |
| build:    | changes to build process                            | none                    | No                        |
| ci:       | changes to CI configuration                         | none                    | No                        |
| style:    | formatting, missing semi-colons, etc                | none                    | No                        |
| chore:    | updating grunt tasks etc; no production code change | none                    | No                        |

</details>

### CODE REVIEW

File: .github/workflows/flow-performance-test.yaml [hunk 1]
```diff
@@ -200,3 +200,71 @@ jobs:
...
+            libgbm1
+
+      - name: Save Screenshots of Metrics Plotter
+        run: |
+          cd ./scripts/metrics-plotter
+          SCREENSHOT_WIDTH=1920 npm run screenshot -- http://127.0.0.1:8123\?hide-controls=true\&grouped=true\&data=timeline ~/.solo/metrics/chart-grouped.png
+          SCREENSHOT_WIDTH=1920 npm run screenshot -- http://127.0.0.1:8123\?hide-controls=true\&grouped=false\&data=timeline ~/.solo/metrics/chart.png
+          cp ./src/data/timeline.json ~/.solo/metrics/timeline-metrics.json
+
+      - name: Stop Metrics Plotter tool
```
Comment (jeromy-cannon) [inline .github/workflows/flow-performance-test.yaml]:
```suggestion
      - name: Stop Metrics Plotter Tool
```
